src/request/export/ExportFile.js

```diff
@@ -31,7 +31,7 @@ class ExportFile extends Nimiq.Observable {
 
         /** @type {HTMLFormElement} */
         const $downloadKeyFilePassphraseBox = ($exportFilePage.querySelector('.passphrase-box'));
-        /** @type {HTMLFormElement} */
+        /** @type {HTMLAnchorElement} */
         const $downloadKeyFile = ($exportFilePage.querySelector('.download-key-file'));
```

Comment:
I deliberately did not rename the variables here, because Sebastian is currently working on this file, and I did not want to change more than I had to.